Resources/public/templates/dashboard/allcontent-dashboard-block.hbt

```diff
@@ -0,0 +1,36 @@
+<h3 class="ez-allcontent-block-title">All Content</h3>
```

Comment:
Should the module titles be treaten as page subtitles?
The reason I put module title in `h3` is that module title is less imporant semantically  than page title or content title.